### PR TITLE
Fix readme consistency issues

### PR DESCRIPTION
**Pull request by nemanjan00**
# What's new

There are some readme consistency issues and this pull request fixes some of them

# Verification 

# Checklist (do not modify)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-<img src="https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png" />
+![](https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png)
 
 Welcome to [Flipper Zero](https://flipperzero.one/)'s Firmware repo!
 Our goal is to create nice and clean code with good documentation, to make it a pleasure for everyone to work with.
 
 # Update firmware
 
-<a href="https://update.flipperzero.one/">Get Latest Firmware from Update Server</a>
+[Get Latest Firmware from Update Server](Get Latest Firmware from Update Server)
```
Comment (skotopes) [inline ReadMe.md]: Hi.

Not an image.

File: ReadMe.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 
 [![Discord](https://img.shields.io/discord/740930220399525928.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](http://flipperzero.one/discord)
 
-<img src="https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png" />
+![](https://habrastorage.org/webt/eo/m0/e4/eom0e4btudte7nrhnyic-laiog0.png)
```
Comment (skotopes) [inline ReadMe.md]: Alt: "Show me the code"

File: ReadMe.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ One liner: `./flash_core1_main.sh`
  - Press and hold `← Left` + `↩ Back` for reset 
  - Release `← Left` and keep holding `↩ Back` until blue LED lights up
  - Release `↩ Back`
-<img src="https://habrastorage.org/webt/uu/c3/g2/uuc3g2n36f2sju19rskcvjzjf6w.png" />
+![](https://habrastorage.org/webt/uu/c3/g2/uuc3g2n36f2sju19rskcvjzjf6w.png)
```
Comment (skotopes) [inline ReadMe.md]: Alt: Switch to DFU sequence

**Review by skotopes — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 1 above)
Comment (nemanjan00) [reply ReadMe.md]: I made a mistake and copied text instead of link

File: ReadMe.md (hunk 2 above)
Comment (nemanjan00) [reply ReadMe.md]: Pushed fix for this

File: ReadMe.md (hunk 3 above)
Comment (nemanjan00) [reply ReadMe.md]: Pushed fix for this

**Review by skotopes — APPROVED** [review]
Nice Work.

**Pull request merged**